### ISSUE TITLE
List of links on FreeBMD2 where we should advise opening in a new tab.

### ISSUE THREAD

**Issue by PatReynolds** [FreeUKGen/FreeBMD2#493]
Discuss multimodal instances on a new story on Systemwide.

Note FreeBMD specific links to open on a new page here:
 - [ ] Report a data problem (if same page, could include the information from the previous page). New feature: return user to previous page after submitting.
 - [ ] District information (currently opens in a new tab)
 - [ ] Print page

**Comment by richardofsussex** [comment]
Maybe not quite on-topic, but the 'Printable format' link in the Tools drop-down should open in a new tab, since the resulting page lacks any navigation controls.

**Comment by AlOneill** [comment]
@richardofsussex Good point — we would have to notify all users that this will happen (accessibility requirement).
Opening in a new tab does not happen on other sites for the equivalent page AFAIK.

**Comment by richpomfret** [comment]
To continue reviewing in next documentation meeting.
